compiler/testData/diagnostics/tests/generics/varProjection/setterProjectedOutAssign.fir.kt

```diff
@@ -7,7 +7,7 @@ interface Tr<T> {
 
 fun test(t: Tr<*>) {
     t.v = null!!
-    t.v = <!ASSIGNMENT_TYPE_MISMATCH!>""<!>
-    t.v = <!NULL_FOR_NONNULL_TYPE!>null<!>
+    t.v = <!SETTER_PROJECTED_OUT!>""<!>
```

Comment:
This is reported on `t.v` in FE 1.0. Then they would be `FIR_IDENTICAL`. I believe you should be able to use `SourceElementPositioningStrategies.ASSIGNMENT_LHS`

Reply:
Aha! Thank you!

Reply:
The other test below (`setterProjectedOutNoPlusAssign`) is `FIR_IDENTICAL`, but this one is still not. I saw you updated the positioning strategy above; is this test data not updated?

Reply:
It turns out I need to report the source of the whole assignment instead. Fixed and updated.